src/wrover_status_light/ServiceIntro.md

```diff
@@ -50,15 +57,159 @@ After defining a message and building this project, ROS would convert a srv file
 | Response message   |`ExamplePackage::ExampleService::Response`|`ExamplePackage.srv.ExampleServiceResponse`|
 
 ## Writing Server and Client nodes
-On how to write server and client nodes, see the demo files in the `src` folder for example code and explanations. Come back to this file and continue when you are done going through them. 
+Inside this module are two demo code files in the `src` folder, one in Python and one in C++. The server code creates a service server node that takes in an interger and doubles it. The client code creates a service client node that uses the server to double a number on repeat. Note that since the service uses `int64` (a integer with 64 bits), running the client code will eventually reach an integer overflow. 
 
-## Running Server and Client nodes
-### Python
+Read through the demo files to understand how to write server and client nodes. Come back to this file and continue when you are ready.  
+
+### C++ Server
+A basic server node can be initialized in C++ with:
+``` C++
+ros::init(argc, argv, "double_server");
+ros::NodeHandle nh;
+ros::ServiceServer service = nh.advertiseService("double_service", doubleServiceCallback);
+ros::spin();
+
+bool doubleServiceCallback(wrover_status_light::DoubleServiceRequest& req, wrover_status_light::DoubleServiceResponse& res) {
```

Comment:
In C++ this would have to be declared before it used in the call to nh.advertiseService(...). For this reason, consider moving this function definition to the top of the example.

Reply:
Addressed in [c0595c3](https://github.com/WisconsinRobotics/WRoverPlayground/pull/9/commits/c0595c34d291b4b911ab00f2ad07d58d9c26804b).